lib/qs/payload_handler.rb

```diff
@@ -43,6 +44,9 @@ def run!(daemon_data, redis_item)
 
       route.run(job, daemon_data)
       redis_item.finished = true
+    rescue DatWorkerPool::ShutdownError => exception
+      handle_exception(exception, daemon_data, redis_item)
```

Comment:
@jcredding we discussed only running this error handler if the item has been started.  This covers a super small edge-case.